Makefile

```diff
@@ -13,19 +13,38 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-PROTOC_IMAGE=$(shell cat Dockerfile | grep FROM |  cut -d' ' -f2)
+PROTOC_IMAGE=protobuf-specs-build
 
 # generate all language protobuf code
-all: go
+all: go python
 
 # generate Go protobuf code
-go:
+go: docker-image
 	@echo "Generating go protobuf files"
-	docker run --pull always --platform linux/amd64 -v ${PWD}:/defs ${PROTOC_IMAGE} -d protos -l go --go-module-prefix github.com/sigstore/protobuf-specs/gen/pb-go
+	docker run --platform linux/amd64 -v ${PWD}:/defs ${PROTOC_IMAGE} -d protos -l go --go-module-prefix github.com/sigstore/protobuf-specs/gen/pb-go
+
+python: docker-image
+	@echo "Generating python protobuf files"
+# we need to manually fix the PYTHONPATH due to: https://github.com/namely/docker-protoc/pull/356
```

Comment:
Ran into an ugly issue which I've got a fix for here: https://github.com/namely/docker-protoc/pull/356

Basically, the buggy PYTHONPATH causes the interpreter to look in the current directory for modules. And since we have a module called `io` at the top level of the `sigstore_protobuf_specs`, the interpreter actually imports that instead the standard library `io` on startup.